modules/nixos.nix

```
@@ -165,5 +164,42 @@ in {
           concatStringsSep " " ruleString
       ) (filter (f: f.enable && f.source != null) (attrValues files));
     }) (filterAttrs (_: u: u.files != {}) config.homes);
+    systemd.services = mapAttrs' (name: {files, ...}: {
+      name = "hjem-monitor-" + name;
+      value = {
+        enable = true;
+        wants = ["systemd-tmpfiles-setup.service" "nix-daemon.socket"];
+        after = ["nix-daemon.socket"];
+        wantedBy = ["default.target"];
+        description = "Monitoring for Hjem files";
+        scriptArgs = "${toString (map (
+          file: file.target
+        ) (filter (f: f.enable && f.source != null) (attrValues files)))}";
+        script = ''
+          #! ${pkgs.runtimeShell} -e
```

```suggestion
				enableStrictShellChecks = true;
        script = ''
          #! ${pkgs.runtimeShell} -e
```

As per eclairevoyant's request for changes. This has not been addressed in recent commits.

---

Added the strict checks